### DISCUSSION TITLE
Discussion notification on the NHP-MRI mailing list

### DISCUSSION THREAD

**Discussion by pcklink** [PRIME-RE/prime-re.github.io#78] [Ideas]
Hi All,

After a discussion at the GCW about the built-in PRIME-RE Discussion site (a Github feature) I think I have now set up the primate.resource.exchange@gmail account to get notifications of posts in the discussion and forward it to the mailing list.

I am not sure if it will do this for all posts or only for new topics, but if the use of the Discussion Site takes off and the number of emails to the mailing lists gets too large, we can re-evaluate this configuration. For existing topics you can always 'subscribe' (see button on the right) so that you will be kept updated.

Let's see if this works as intended.

Best, Chris

**Comment by rhartig** [comment]
Thanks Chris for all your efforts on this!
